### PR TITLE
Dev yaseen

### PR DESCRIPTION
**Pull request by yadallash**
Added car makes to fireBase, AuthenticationService and DataBaseService.

### CODE REVIEW

File: SpotSwap/Models/EndUser.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+//  EndUser.swift
+//  SpotSwap
+//
+//  Created by Yaseen Al Dallash on 3/14/18.
+//  Copyright © 2018 Yaseen Al Dallash. All rights reserved.
+//
+
+import Foundation
+
+struct EndUser: Codable {
```
Comment (SaiKhal) [inline SpotSwap/Models/EndUser.swift]: Is there a better name for this object? We arent sure what an "EndUser" is.

File: SpotSwap/Services/FireBase Services/DataBase Service/DataBaseService+EndUsers.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//
+//  Created by Yaseen Al Dallash on 3/14/18.
+//  Copyright © 2018 Yaseen Al Dallash. All rights reserved.
+//
+
+import Foundation
+import Firebase
+extension DataBaseService{
+    //This will add a new EndUser to the dataBase using the user UID
+    func addNewEndUser(endUser: EndUser, user: User, completion: ()->Void, errorHandler: (Error)->Void){
```
Comment (SaiKhal) [inline SpotSwap/Services/FireBase Services/DataBase Service/DataBaseService+EndUsers.swift]: What is the difference between EndUser and User? Caroline would like you to be more descriptive.

**Review by SaiKhal — CHANGES_REQUESTED** [review]
Very nice but there is some ambuiguity with the Type "EndUser"

**Review by caroline608 — CHANGES_REQUESTED** [review]
Will close soy can commit new changes

**Pull request closed without merging**